Source/Core/BoogiePL.atg

```diff
@@ -520,7 +520,13 @@ Function<.out List<Declaration>/*!*/ ds.>
     if (definition != null) {
       // generate either an axiom or a function body
       if (QKeyValue.FindBoolAttribute(kv, "inline")) {
+        if (QKeyValue.FindBoolAttribute(kv, "define"))
+          SemErr("function cannot have both inline and define attributes");
         func.Body = definition;
+      } else if (QKeyValue.FindBoolAttribute(kv, "define")) {
+        func.DefinitionBody = func.CreateFunctionDefinition(definition);
+        if (func.TypeParameters.Count > 0)
+          SemErr("function with define attribute has to be monomorphic");
```

Comment:
I suggest also using the colon (but not curly braces) to these messages, as in:

    both :inline and :define attributes

It makes it clearer that these are names of attributes, as opposed to general words in the sentence.

Both lines 524 and 529.

Reply:
Good point. Done.